tests/test_helpers.py

```diff
@@ -56,6 +56,8 @@
     (["helper", "l_contacts", "run", "-n", "ny", "-i", "col", "-o", "coffee", "-d", "2019-01-15", "-r", "2"],
      ""
      ),
+    (["helper", "l_appointments", "-g", "mrsec14"],  "f19, scopatz"
```

Comment:
some more things are needed like begin and end date and loading

Reply:
I believe I just added begin and end date, but could you perhaps clarify the "loading" part?